larky/pom.xml

```diff
@@ -222,6 +222,9 @@
             </resource>
             <resource>
                 <directory>src/test/resources</directory>
+                <excludes>
+                    <exclude>META-INF/services/*</exclude>
+                </excludes>
```

Comment:
Why do we need to explicitly exclude it?

Reply:
@aslepakurov The files in this directory are used to select an SPI provider. We need this directory available in testing, but it cannot be included in the package otherwise it won't be overridable. So I exclude it.